Tools/TestWebKitAPI/Tests/WebCore/curl/CurlMultipartHandleTests.cpp

```diff
@@ -139,7 +139,7 @@ TEST(CurlMultipartHandleTests, SimpleMessage)
     auto curlResponse = createCurlResponse();
     auto handle = CurlMultipartHandle::createIfNeeded(client, curlResponse);
 
-    handle->didReceiveMessage(SharedBuffer::create(data, strlen(data)));
+    handle->didReceiveMessage({ reinterpret_cast<const uint8_t*>(data.characters()), data.length() });
```

Comment:
How about defining an function converting a `ASCIILiteral` to `std::span<const uint8_t>` in this file?
std::span<const uint8_t> toSpan(const ASCIILiteral& x);

Reply:
Fixed